File: 283-MoveZeroes/283-MoveZeroes.py
# Last updated: 8/13/2025, 8:20:22 PM
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        j=0
        for i in range (len(nums)) :
            if nums[i]!=0 :
                [nums[i] , nums[j]] = [nums[j] , nums[i]]
                j+=1

File: 283-MoveZeroes/283-MoveZeroes.java
// Last updated: 8/13/2025, 8:20:36 PM
class Solution {
    public void moveZeroes(int[] nums) {
        int i = 0;
        for(int j=0;j<nums.length;j++){
            if(nums[j]!=0){
                int temp = nums[j];
                nums[j] = nums[i];
                nums[i] = temp;
                i++;
            }
        }
    }
}